spring/boot-autoconfigure/src/test/java/com/linecorp/armeria/spring/ArmeriaAutoConfigurationTest.java

```diff
@@ -105,11 +116,41 @@ protected HttpResponse doGet(ServiceRequestContext ctx, HttpRequest req) throws
         }
     }
 
+    public static class IllegalArgumentExceptionHandler implements ExceptionHandlerFunction {
+
+        @Override
+        public HttpResponse handleException(RequestContext ctx, HttpRequest req, Throwable cause) {
+            if (cause instanceof IllegalArgumentException) {
+                return HttpResponse.of(HttpStatus.OK, MediaType.PLAIN_TEXT_UTF_8, "exception");
+            }
+            return ExceptionHandlerFunction.fallthrough();
+        }
+    }
+
+    public static class StringResponseConverter implements ResponseConverterFunction {
+        @Override
+        public HttpResponse convertResponse(ServiceRequestContext ctx, @Nullable Object result)
+                throws Exception {
+            if (result instanceof String) {
+                return HttpResponse.of(HttpStatus.OK,
+                                       MediaType.ANY_TEXT_TYPE,
+                                       HttpData.ofUtf8(result.toString()));
```

Comment:
Could be shorten to `HttpResponse.of(MediaType.ANY_TEXT_TYPE, result.toString())`